File: Server/Models/Expense.js
const mongoose = require('mongoose')
const Schema = mongoose.Schema

const ExpenseSchema = new Schema ({
    objectID:{
        type: String,
        required: [true,' Object name is required'],
        trim: true,
    },
    name:{
        type: String,
        required: [true,' Name is required'],
        trim: true,
    },
    creatorID:{
        type: String,
        required: [true,'Creator ID is required'],
        trim: true,
    },
    amount:{
        type: Number,
        required: [true,' Amount is required'],
    },
    description:{
        type: String,
        trim: true,
        minlength: 1,
        maxlength: 100,
        required: [true, 'Description is required']
    },
    expenseDate: Date,
    expenseTime: Time,
    resolverID:{
        type: String,
        trim: true,
    }
})

const Expense = mongoose.model('Expense', ExpenseSchema)

module.exports = Expense

File: hello_world/lib/Models/expense.dart
class ExpenseModel{
  String name;
  String? objectID;
  String? creatorID;
  String amount;
  String description;
  DateTime? date;

  ExpenseModel({required this.name, required this.objectID ,required this.creatorID,required this.amount,required this.description,required this.date});

  factory ExpenseModel.fromJson(Map<String, dynamic> json){
    return ExpenseModel(
      name: json['Name'],
      objectID: json['ObjectID'],
      creatorID: json['CreatorID'],
      amount: json['Amount'],
      description: json['Description'],
      date: json['Date'].toDate(),
    );
  }

  Map<String, dynamic> toJson() => {
    'Name': name,
    'ObjectID': objectID,
    'CreatorID': creatorID,
    'Amount': amount,
    'Description': description,
    'Date': date,
  };
}